App-Auto-Patch-via-Dialog.zsh

```diff
@@ -163,6 +163,8 @@ set_defaults() {
 
     installomatorOptions="BLOCKING_PROCESS_ACTION=prompt_user NOTIFY=silent LOGO=appstore" # MDM Enabled
 
+    installomatorVersion="Release" # MDM Enabled - Use:  Release|Beta 
```

Comment:
Against all odds i really would like the default to be changed to "Beta". You may ask why? Well AAP is designed to be run often to get the latest versions installed, this requires the label from Installamator to be a.) working and b.) updated to what ever the vendor has changed. Given the current release cycle Installomator has, no blaming here, there whole purpose of having a automated patching service like AAP would be contradicted using old and not-working installomatore labels.

for nearly any other software in a corp envornment or even at home i would prefer "stable" / "released" software versions.

also i ask why the Intune MDM Config is already defaulting to `Beta` the script itself here is not.